### PR TITLE
feat: add k9s completion spec

### PR DESCRIPTION
**Pull request by AlanSyue**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
feature

**What is the current behavior? (You can also link to an open issue here)**
N/A

**What is the new behavior (if this is a feature change)?**
Add `k9s` completion spec ([issue](https://github.com/withfig/autocomplete/issues/1454))

**Additional info:**
N/A

### CODE REVIEW

File: src/k9s.ts [hunk 1]
```diff
@@ -0,0 +1,199 @@
+const completionSpec: Fig.Spec = {
+  name: "k9s",
+  description:
+    "K9s is a terminal based UI to interact with your Kubernetes clusters",
+  subcommands: [
+    {
+      name: "help",
+      description: "Help about any command",
+    },
```
Comment (SeparateRecords) [inline src/k9s.ts:9]:
```suggestion
    {
      name: "help",
      description: "Help about any command",
      args: { name: "command", isOptional: true, template: "help" },
    },
```

File: src/k9s.ts [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    {
+      name: ["-l", "--logLevel"],
+      description:
+        "Specify a log level (info, warn, debug, trace, error) (default 'info')",
+      args: {
+        name: "string",
+        description: "The log level",
+        default: "info",
+      },
```
Comment (SeparateRecords) [inline src/k9s.ts:130]:
```suggestion
      args: {
        name: "string",
        description: "The log level",
        default: "info",
        suggestions: ["info", "warn", "debug", "trace", "error"],
      },
```

**Review by SeparateRecords — DISMISSED** [review]
Looking good!! Not much to change here, thanks so much for contributing this 🙂

File: src/k9s.ts [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+        name: "string",
+        description: "The path to a cert file",
+      },
+    },
+    {
+      name: "--client-key",
+      description: "Path to a client key file for TLS",
+      args: {
+        name: "string",
+        description: "The path to a client key file",
```
Comment (fedeci) [inline src/k9s.ts:70]:
```suggestion
        description: "The path to a client key file",
        template: "filepaths",
```

File: src/k9s.ts [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+        name: "stringArray",
+        description: "The group you want to impersonate",
+      },
+    },
+    {
+      name: "--certificate-authority",
+      description: "Path to a cert file for the certificate authority",
+      args: {
+        name: "string",
+        description: "The path to a cert file",
```
Comment (fedeci) [inline src/k9s.ts:62]:
```suggestion
        description: "The path to a cert file",
        template: "filepaths",
```

File: src/k9s.ts [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+      name: "--insecure-skip-tls-verify",
+      description:
+        "If true, the server's caCertFile will not be checked for validity",
+    },
+    {
+      name: "--kubeconfig",
+      description: "Path to the kubeconfig file to use for CLI requests",
+      args: {
+        name: "string",
+        description: "The path to the kubeconfig file",
```
Comment (fedeci) [inline src/k9s.ts:112]:
```suggestion
        description: "The path to the kubeconfig file",
        template: "filepaths",
```

File: src/k9s.ts [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+      },
+    },
+    {
+      name: "--logFile",
+      description: "Specify the log file",
+      args: {
+        name: "string",
+        description: "The path of the log file",
```
Comment (fedeci) [inline src/k9s.ts:120]:
```suggestion
        description: "The path of the log file",
        template: "filepaths",
        suggestCurrentToken: true,
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks for this contribution @AlanSyue! Let me know if you need any help!

**Comment by AlanSyue** [comment]
> Thanks for this contribution @AlanSyue! Let me know if you need any help!

Thanks for your suggestion!

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**